src/Compilers/VisualBasic/Portable/Symbols/ParameterSymbol.vb

```diff
@@ -65,6 +65,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             End Get
         End Property
 
+        Friend ReadOnly Property IsDiscard As Boolean Implements IParameterSymbol.IsDiscard
```

Comment:
>Friend ReadOnly Property IsDiscard As Boolean Implements IParameterSymbol.IsDiscard [](start = 8, length = 83)

Please move this method to the ```#Region "IParameterSymbol"``` region below and follow the same pattern for naming and accessibility as for other APIs in that region.